deploy/1_seed.ts

```diff
@@ -0,0 +1,16 @@
+import { Logger } from 'ethers/lib/utils';
+import { validateDeployment, configureDeploymentSettings, seedContracts } from '../utils/deploy';
+import { LogLevel } from '@ethersproject/logger';
+import { getContractsFromDeployments } from '../test/helpers/index';
```

Comment:
```suggestion
import { getContractsFromDeployments } from '@/test/helpers';
```

Reply:
we should also remove this from the test directory since we use it outside of tests